terraform/modules/compute_plane/locals.tf

```diff
@@ -10,7 +10,8 @@ locals {
     "InitWorker__WorkerCheckRetries=${var.polling_agent.worker_check_retries}",
     "InitWorker__WorkerCheckDelay=${var.polling_agent.worker_check_retries}",
     "Zipkin__Uri=${var.zipkin_uri}",
-    "Amqp__PartitionId=TestPartition${local.partition_chooser}"
+    "Amqp__PartitionId=TestPartition${local.partition_chooser}",
+    "DependencyResolver__UnresolvedDependenciesQueue=${var.unresolved_dependencies_queue}"
```

Comment:
Add a final comma

Reply:
We will need to check terraform files formatting and style